src/App.tsx

```diff
@@ -20,12 +30,15 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={searchMovie}
+                onChange={handleChange}
               />
             </div>
           </div>
         </div>
-
-        <MoviesList movies={moviesFromServer} />
+        {visibleMovies
+          ? <MoviesList movies={visibleMovies} />
+          : <MoviesList movies={moviesFromServer} />}
```

Comment:
strange logic: if no films were found - user will see all films? I think it's better to show only visible movies